biome.json

```diff
@@ -6,7 +6,8 @@
   "formatter": {
     "enabled": true,
     "indentWidth": 2,
-    "indentStyle": "space"
+    "indentStyle": "space",
+    "ignore": ["package.json"]
```

Comment:
We might be formatting during deploy. Ignoring `package.json` for now.

Reply:
Nice